### PR TITLE
fix: update conditions

### PR DESCRIPTION
**Pull request by qasimqlf**

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_whoami_priv.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ references:
     - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/whoami
 author: Florian Roth (Nextron Systems)
 date: 2021/05/05
-modified: 2022/05/13
+modified: 2023/02/03
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_whoami_priv.yml:9]:
```suggestion
modified: 2022/05/13
```
we shouldn't change the modified date for cosmetic changes

File: rules/windows/process_creation/proc_creation_win_tool_nircmd.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ references:
     - https://www.nirsoft.net/utils/nircmd2.html#using
 author: Florian Roth (Nextron Systems), Nasreddine Bencherchali
 date: 2022/01/24
-modified: 2022/11/30
+modified: 2023/02/03
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_tool_nircmd.yml:11]:
```suggestion
modified: 2022/11/30
```
we shouldn't change the modified date for cosmetic changes

File: rules/windows/process_creation/proc_creation_win_termserv_proc_spawn.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ references:
     - https://securingtomorrow.mcafee.com/other-blogs/mcafee-labs/rdp-stands-for-really-do-patch-understanding-the-wormable-rdp-vulnerability-cve-2019-0708/
 author: Florian Roth (Nextron Systems)
 date: 2019/05/22
-modified: 2023/01/25
+modified: 2023/02/03
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_termserv_proc_spawn.yml:9]:
```suggestion
modified: 2023/01/25
```
we shouldn't change the modified date for cosmetic changes

File: rules/windows/process_creation/proc_creation_win_regedit_export_critical_keys.yml [hunk 4]
```diff
@@ -10,29 +10,31 @@ references:
     - https://gist.github.com/api0cradle/cdd2d0d0ec9abb686f0e89306e277b8f
 author: Oddvar Moe, Sander Wiebing, oscd.community
 date: 2020/10/12
-modified: 2022/07/07
+modified: 2023/02/03
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_regedit_export_critical_keys.yml:13]:
```suggestion
modified: 2022/07/07
```
we shouldn't change the modified date for cosmetic changes

File: rules/windows/process_creation/proc_creation_win_python_pty_spawn.yml [hunk 5]
```diff
@@ -9,25 +9,26 @@ references:
     - https://www.volexity.com/blog/2022/06/02/zero-day-exploitation-of-atlassian-confluence/
 author: Nextron Systems
 date: 2022/06/03
+modified: 2023/02/03
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_python_pty_spawn.yml:12]:
```suggestion
```
we shouldn't change the modified date for cosmetic changes

File: rules/windows/process_creation/proc_creation_win_regedit_import_keys_ads.yml [hunk 6]
```diff
@@ -10,20 +10,22 @@ references:
...
-        Image|endswith: '\regedit.exe'
+    selection_img:
+        - Image|endswith: '\regedit.exe'
+        - OriginalFileName: 'REGEDIT.EXE' 
+    selection_cli:
         CommandLine|contains:
             - ' /i '
             - '.reg'
-    selection_2:
+    selection_cli_re:
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_regedit_import_keys_ads.yml:28]:
```suggestion
```
no need for the  extra selection. We can use a key multiple times when the value modifiers differ.

**Review by phantinuss — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_python_pty_spawn.yml (hunk 5 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_python_pty_spawn.yml:12]: @qasimqlf please update this

**Review by nasbench — APPROVED** [review]

**Pull request merged**